package/boot/uboot-envtools/files/qualcommax_ipq807x

```diff
@@ -27,6 +27,13 @@ edimax,cax1800)
 	[ -n "$idx" ] && \
 		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x10000" "0x20000"
 	;;
+linksys,mx4200v1|\
+linksys,mx4200v2)
+	idx="$(find_mtd_index u_env)"
+	[ -n "$idx" ] && \
+		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x40000" "0x20000" "2"
+	;;
+
```

Comment:
IPQ8173/IPQ8174 is marketed as Qualcomm Pro 800 Networking platform, not sure if we want to combine with ipq807x

There are hardware differences between the 2, as provided by Qualcomm, though the cipset is the same 

https://www.qualcomm.com/products/internet-of-things/networking/wi-fi-networks/ipq8074
https://www.qualcomm.com/products/internet-of-things/networking/wi-fi-networks/networking-pro-series/qualcomm-networking-pro-800-platform

I would suggest to add a new platform onboarding for IPQ8174, rather than adding to ipq807x.

Reply:
No, they are functionally the same when it comes to the software.
The main difference is CPU and NSS clocks, like with IPQ8070/1A.

Reply:
yeah could be done later when theres other devices with this chipset added

Reply:
There is really no point for Oak specific subtarget

Reply:
> There is really no point for Oak specific subtarget

yeah after reading your comment, it'd be a waste of time todo it